src/style.css

```diff
@@ -1 +1,79 @@
-/* Styles go here */
+:root {
+  --colorForLinks: #00acdc;
+  font-family: "Roboto", Arial, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.menu {
+  padding: 0 50px;
+}
+
+.menu__content {
+  max-width: 1200px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 auto;
+}
+
+.MOYO__logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+  padding: 10px 0;
+}
+
+.nav__list {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__box {
+  margin-right: 20px;
+}
+
+.nav__box:last-child {
```

Comment:
You can use `.nav__box:not(:last-child)` selector to not apply margin-right to the last element